src/tasks/component-installers/cert-manager.ts

```diff
@@ -65,6 +67,8 @@ export class CertManagerTasks {
             throw new Error('Cert Manager should be deployed before.')
           }
 
+          ctx.certManagerK8sApiVersion = await this.kubeHelper.getCertManagerK8sApiVersion()
```

Comment:
We should move this to `Check Cert Manager deployment` task due to `enabled: ctx => ctx.certManagerInstalled,`

Reply:
Cert Manager will be present at that step. The check is done just for case.